vpp/vaapipostprocess_scaler_unittest.cpp

```diff
@@ -255,4 +255,36 @@ VAAPIPOSTPROCESS_SCALER_TEST(Deinterlace)
 {
     vppTest(VppDeinterlace);
 }
+
+void VppClrBalance(VaapiPostProcessScaler& scaler, const SharedPtr<VideoFrame>& src, const SharedPtr<VideoFrame>& dest)
+{
+    VPPColorBalanceParameter params;
+    memset(&params, 0, sizeof(params));
+    params.size = sizeof(params);
+
+    params.mode = COLORBALANCE_HUE;
+    params.level = 50;
+    EXPECT_EQ(YAMI_SUCCESS, scaler.setParameters(VppParamTypeColorBalance, &params));
+    EXPECT_EQ(YAMI_SUCCESS, scaler.process(src, dest));
+
+    params.mode = COLORBALANCE_SATURATION;
+    params.level = 50;
+    EXPECT_EQ(YAMI_SUCCESS, scaler.setParameters(VppParamTypeColorBalance, &params));
+    EXPECT_EQ(YAMI_SUCCESS, scaler.process(src, dest));
+
+    params.mode = COLORBALANCE_BRIGHTNESS;
+    params.level = 50;
+    EXPECT_EQ(YAMI_SUCCESS, scaler.setParameters(VppParamTypeColorBalance, &params));
+    EXPECT_EQ(YAMI_SUCCESS, scaler.process(src, dest));
+
+    params.mode = COLORBALANCE_CONTRAST;
+    params.level = 50;
+    EXPECT_EQ(YAMI_SUCCESS, scaler.setParameters(VppParamTypeColorBalance, &params));
+    EXPECT_EQ(YAMI_SUCCESS, scaler.process(src, dest));
+}
+
+VAAPIPOSTPROCESS_SCALER_TEST(Clrbalance)
```

Comment:
Can you rename this test "ColorBalance" instead... the "Clr" part is ambiguous.  For test names, verbosity is better.